www/docs/further/further-reading.md

```diff
@@ -49,5 +49,9 @@ Blitz is a full-stack framework & achieves this by maintaining a fork of Next.js
 ### Benefits with Blitz.js
 
 - Data layer integrated with the Next.js runtime, resulting in a more integrated experience
-- Automatic integration with Blitz' auth
+- Automatic integration with Blitz's auth
```

Comment:
Oh, I've always treated "z" at the end as an "s" :)

Reply:
Oh interesting! Never seen that convention before but it seems it's not unheard of.

> [Rule](https://www.grammarbook.com/blog/apostrophes/apostrophes-with-names-ending-in-s-ch-or-z/): To show singular possession of a name ending in s or z, some writers add just an apostrophe. Others also add another s. See Rules 1b and 1c of [Apostrophes](https://www.grammarbook.com/punctuation/apostro.asp) for more discussion.

Also: [Apostrophe after Z (with Examples)](https://www.grammar-monster.com/punctuation/apostrophe_after_z.html).

For me I think it reads a little cleaner but would be fine changing it back.

Reply:
English isn't my first language so sometimes I just wing it 🕺